scipy/interpolate/tests/test_fitpack.py

```diff
@@ -79,7 +79,7 @@ def check_1(self,f=f1,per=0,s=0,a=0,b=2*pi,N=20,at=0,xb=None,xe=None):
             xe = b
         x = a+(b-a)*arange(N+1,dtype=float)/float(N)    # nodes
         x1 = a+(b-a)*arange(1,N,dtype=float)/float(N-1)  # middle points of the nodes
-        v,v1 = f(x),f(x1)
+        v, _ = f(x),f(x1)
```

Comment:
If f(x1) is not used, maybe just avoid computing it

Reply:
Done.